src/Rx.KitchenSink.ts

```diff
@@ -8,6 +8,8 @@ import {Scheduler as IScheduler} from './Scheduler';
 export interface KitchenSinkOperators<T> extends CoreOperators<T> {
   isEmpty?: () => Observable<boolean>;
   elementAt?: (index: number, defaultValue?: any) => Observable<T>;
+  distinct?: (compare?: (x: T, y: T) => boolean, flushes?: Observable<any>) => Observable<T>;
```

Comment:
This one should be in KitchenSink since it's not essential. Do I need to fix this?